packages/assistant/src/RPA/Assistant/flet_client.py

```diff
@@ -137,8 +147,30 @@ def add_element(
     ):
         # TODO: validate that element "name" is unique
         # make a container that adds margin around the element
-        container = Container(margin=5, content=element)
-        self._elements.visible.append(container)
+
+        # if it's a Container we don't create our own margin container to allow setting
+        # of padding and absolute position manually by user
+        if isinstance(element, Container):
+            new_element = element
+        else:
+            new_element = Container(margin=5, content=element)
+
+        if self._layout_stack != []:
+            current_layout = self._layout_stack[-1]
+            if hasattr(current_layout, "controls"):
+                # Row, Stack and Column have "controls"
+                current_layout.controls.append(new_element)
+            elif hasattr(current_layout, "actions"):
```

Comment:
wouldn't `isinstance` be safer here?

Reply:
True. Actually we should do both `hasattr` and `isinstance` check, so that our tests will easily catch if the Flet API regarding these classes has an update that removes these properties.

Reply:
fixed to isinstance.